#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# (c) ACE 

import os

class Config(object):
    # get a token from @BotFather
    pass
    """
    BOT_TOKEN = os.environ.get("BOT_TOKEN", "7117502279:AAHk5S387m1mTl4DLcUEekGTciYCN7GLbrU")
    API_ID = int(os.environ["API_ID", 26484988]
    API_HASH = os.environ["API_HASH", "8d6e625fe3e296b4b6bd4817aca39ec5"]
    AUTH_USERS = "6888054311"""
# jisko bhi is repo ko deploy karne me error aa raha ho ( https://t.me/Stubborn1223 ) ye mere TG ka id ha message karke iska screen shot send kar dena me error solve kar dunga. 
